src/components/logic/always_after.py

```diff
@@ -0,0 +1,40 @@
+""" Algorithm to detect the always after relationships in a given log """
+import os
+from src.components.logic.relationship import Relationship
+from src.components.util.xes_importer import XES_Importer
+from pm4py.algo.discovery.log_skeleton import algorithm as lsk_discovery
+
+
+class AlwaysAfter(Relationship):
+    """ Implementation of the always after relationship"""
+
+    def __init__(self, traces):
+        super().__init__(traces)
```

Comment:
Is the Linter not asking for a docstring here?

Reply:
Apparently not, otherwise the test would not pass.

Reply:
I was about to change it haha, also pycharm usually shows all Linter warnings, but in this case it didnt show anything